presets/inference/text-generation/inference_api.py

```diff
@@ -81,10 +82,37 @@ def __post_init__(self):
 model_args = asdict(args)
 model_args["local_files_only"] = not model_args.pop('allow_remote_files')
 model_pipeline = model_args.pop('pipeline')
+adapters_name_or_path = model_args.pop('adapters_name_or_path')
 
 app = FastAPI()
 tokenizer = AutoTokenizer.from_pretrained(**model_args)
-model = AutoModelForCausalLM.from_pretrained(**model_args)
+base_model = AutoModelForCausalLM.from_pretrained(**model_args)
+
+def parse_adapter_paths(adapter_paths_str):
+    adapter_paths_list = adapter_paths_str.split(',')
+    return adapter_paths_list
+
+adapters_list = parse_adapter_paths(adapters_name_or_path)
+adapters_list= [adapter.strip() for adapter in adapters_list if adapter.strip()]
+
+if len(adapters_list) == 0:
+    model = base_model
+else: 
+    model = PeftModel.from_pretrained(base_model, adapters_list[0], adapter_name="adapter-0")
+    for i in range(1, len(adapters_list)):
+        model.load_adapter(adapters_list[i], "adapter-"+str(i))
+    adapters, weights= [], []
+    for i in range(0, len(adapters_list)):
```

Comment:
Can you add a comment here about what are supposed to be added to the env? and what if the adapter_num does not exist in the env?

Reply:
Only strength in the env, if the users enter  no strength, we set a default value for it